server/src/main/java/io/spine/server/aggregate/Aggregate.java

```diff
@@ -124,12 +124,14 @@
 public abstract class Aggregate<I,
                                 S extends EntityState<I>,
                                 B extends ValidatingBuilder<S>>
-        extends CommandHandlingEntity<I, S, B>
+        extends CommandAssigningEntity<I, S, B>
```

Comment:
I would rename this one to `CommandAssigneeEntity`. It's because "command-assigning entity" is the one which performs the assignment, and not the one which _is_ assigned. And we mean the latter, not the former.